### PR TITLE
chore: polish kb3-kb13 files

### PR DESCRIPTION
**Pull request by Meng-Heng**

### CODE REVIEW

File: knowledge-base/kb0012.md [hunk 1]
```diff
@@ -1,58 +1,50 @@
...
+  - Click Next, then Finish, then click Yes to install the certificate.
+  - Repeat steps 5-8 for each of the other two certificates.
+  - Finally, open the folder "Thawte Server Roots", and follow steps 5-8 for the files 
+   "ThawtePremiumServerCA.cer" and
+  "ThawteServerCA.cer".  These files have an icon that looks like a certificate.
+
+- If you are running Keyman 6.2.176.0 or later, or version 7.0, the following procedure will usually correct corrupted Root Certificates:
+  - Visit the following website:
+    ["http://www.verisign.com/support/roots.html"] 
+    (http://www.verisign.com/support/roots.html)
```
Comment (darcywong00) [inline knowledge-base/kb0012.md:30]: Combine lines so the hyperlink works
```suggestion
["http://www.verisign.com/support/roots.html"](http://www.verisign.com/support/roots.html)
```

**Review by Meng-Heng — COMMENTED** [review]
edited: false message

**Review by Meng-Heng — COMMENTED** [review]
Updated kb12-13

**Review by darcywong00 — APPROVED** [review]
lgtm

I changed the base to "master"

File: knowledge-base/kb0012.md [hunk 2]
```diff
@@ -1,58 +1,49 @@
...
+  - Click "Place all certificates in the following store", click Browse, and select     
+  "Trusted Root Certification Authorities".
+  - Click Next, then Finish, then click Yes to install the certificate.
+  - Repeat steps 5-8 for any of the other certificates that appear to be Class 3 
+   certificates.
+
+- You may also need to re-register some or all of the following system files depending on your version of Windows: softpub.dll, wintrust.dll, initpki.dll, ssenh.dll, saenh.dll, gpkcsp.dll, sccbase.dll, slbcsp.dll, mssip32.dll, cryptdlg.dll To do this:
+  - Run the command prompt ('command' in Windows 98, 'cmd' in Windows 2000/XP)
+  - Enter the following for each of the above files: **regsvr32 _dll name_** where _dll 
+   name_ is the name of the dll to re-register.
```
Comment (Meng-Heng) [inline knowledge-base/kb0012.md:45]:
```suggestion
1. Check that your system is free of viruses and spyware.  Both of these can cause problems with installing Keyman.
2. Ensure you have enough disk space to install Keyman.
3. If you receive an error about Keyman being corrupt, your certificates that verify software authenticity may have
been corrupted or deleted.  This is usually caused by spyware (see point 1 above).
4. If you are running Keyman 6.2.175.0 or earlier, the following procedure will usually
correct corrupted Root Certificates:
  i. Visit the following website: 
  http://www.thawte.com/roots/
  ii. After filling in the form there, you will download a file called thawte-roots.zip.
  iii. You must first extract this file to a temporary folder (for example, using WinZip).
  iv. In the temporary folder, you will find 3 certificate files and 2 folders.
  v. Double-click on the first certificate file in the folder.
  vi. Click the Install Certificate button, then click Next.
  vii. Click "Place all certificates in the following store", click Browse, and select "Trusted Root Certification Authorities".
  viii. Click Next, then Finish, then click Yes to install the certificate.
  ix. Repeat steps 5-8 for each of the other two certificates.
  x. Finally, open the folder "Thawte Server Roots", and follow steps 5-8 for the files "ThawtePremiumServerCA.cer" and "ThawteServerCA.cer". These files have an icon that looks like a certificate.
5. If you are running Keyman 6.2.176.0 or later, or version 7.0, the following procedure will usually correct corrupted Root Certificates:
  i. Visit the following website:
[http://www.verisign.com/support/roots.html](http://www.verisign.com/support/roots.html)
  ii. After filling in the form there, you will download a file called roots.zip.
  iii. You must first extract this file to a temporary folder (for example, using WinZip).
  iv. In the temporary folder, you will find several .cer certificate files. We are interested in class 3 (e.g. C3_PCA_G3v2.cer or similar).
  v. Double-click on the certificate file in the folder.
  vi. Click the Install Certificate button, then click Next.
  vii. Click "Place all certificates in the following store", click Browse, and select "Trusted Root Certification Authorities".
  viii. Click Next, then Finish, then click Yes to install the certificate.
  ix. Repeat steps 5-8 for any of the other certificates that appear to be Class 3 certificates.

6. You may also need to re-register some or all of the following system files depending on your version of Windows: softpub.dll, wintrust.dll, initpki.dll, ssenh.dll, saenh.dll, gpkcsp.dll, sccbase.dll, slbcsp.dll, mssip32.dll, cryptdlg.dll To do this:
  i. Run the command prompt ('command' in Windows 98, 'cmd' in Windows 2000/XP)
  ii. Enter the following for each of the above files: **regsvr32 _dll name_** where _dll name_ is the name of the dll to re-register.

```

**Review by Meng-Heng — COMMENTED** [review]
correcting the ordered lists

File: knowledge-base/kb0012.md [hunk 3]
```diff
@@ -1,58 +1,41 @@
...
+  http://www.thawte.com/roots/
+  ii. After filling in the form there, you will download a file called thawte-roots.zip.
+  iii. You must first extract this file to a temporary folder (for example, using WinZip).
+  iv. In the temporary folder, you will find 3 certificate files and 2 folders.
+  v. Double-click on the first certificate file in the folder.
+  vi. Click the Install Certificate button, then click Next.
+  vii. Click "Place all certificates in the following store", click Browse, and select "Trusted Root Certification Authorities".
+  viii. Click Next, then Finish, then click Yes to install the certificate.
+  ix. Repeat steps 5-8 for each of the other two certificates.
+  x. Finally, open the folder "Thawte Server Roots", and follow steps 5-8 for the files "ThawtePremiumServerCA.cer" and "ThawteServerCA.cer". These files have an icon that looks like a certificate.
```
Comment (Meng-Heng) [inline knowledge-base/kb0012.md:21]:
```suggestion
  1. Visit the following website: 
  http://www.thawte.com/roots/
  2. After filling in the form there, you will download a file called thawte-roots.zip.
  3. You must first extract this file to a temporary folder (for example, using WinZip).
  4. In the temporary folder, you will find 3 certificate files and 2 folders.
  5. Double-click on the first certificate file in the folder.
  6. Click the Install Certificate button, then click Next.
  7. Click "Place all certificates in the following store", click Browse, and select "Trusted Root Certification Authorities".
  8. Click Next, then Finish, then click Yes to install the certificate.
  9. Repeat steps 5-8 for each of the other two certificates.
  10. Finally, open the folder "Thawte Server Roots", and follow steps 5-8 for the files "ThawtePremiumServerCA.cer" and "ThawteServerCA.cer". These files have an icon that looks like a certificate.

```

**Pull request merged**